browserassets/js/browserOutput.js

```diff
@@ -843,6 +859,26 @@ $(function() {
         setCookie('fonttype', font, 365);
     });
 
+    $('#chooseTheme').click(function(e) {
+        if ($('.popup .changeTheme').is(':visible')) {return;}
+        var popupContent = '<div class="head">Change Theme</div><div id="changeTheme" class="changeTheme">';
+        $.each(themes, function(themeclass, themename) {
+          popupContent = popupContent + '<a href="#" data-theme="'+themeclass+'">'+themename+'</a>';
+        })
+
+        popupContent = popupContent + '</div>';
+        createPopup(popupContent, 200);
+    });
+
+    $('body').on('click', '#changeTheme a', function(e) {
+        var theme = $(this).attr('data-theme');
+        var body = $('body');
+        body.removeClass(opts.currentTheme);
+        body.addClass(theme);
```

Comment:
Incredible nitpick, this might as well just be:
`$('body').removeClass(opts.currentTheme).addClass(theme);`